cmd/vela-terraform/main.go

```diff
@@ -292,13 +294,19 @@ func run(c *cli.Context) error {
 		"registry": "https://hub.docker.com/r/target/vela-terraform",
 	}).Info("Vela Terraform Plugin")
 
-	// capture custom terraform version requested
-	version := c.String("terraform.version")
+	// capture custom terraform tfVersion requested
+	tfVersion := c.String("terraform.tfVersion")
```

Comment:
could be misunderstanding but why did we change this to `terraform.tfVersion`. it looks like it's still named `terraform.version` here: https://github.com/go-vela/vela-terraform/blob/b74f9abd3a75ff152c3771d863b5b1f947b2463a/cmd/vela-terraform/main.go#L147-L152

Reply:
I refactored that variable because of a name collision with the import "github.com/go-vela/vela-terraform/version"

Reply:
i think the golang variable rename `version` -> `tfVersion` is not of concern. but the Flag name reference of `terraform.tfVersion` is not accurate anymore, is it?

Reply:
Nice catch, I fixed that.